recipes/ibmz_ci.rb

```diff
@@ -33,16 +33,6 @@ class ::Chef::Recipe
   "client-#{node['fqdn'].tr('.', '-')}"
 )
 
-ruby_block 'Set jenkins username/password if needed' do
-  block do
-    if ::File.exist?('/var/lib/jenkins/config.xml') &&
-       ::File.foreach('/var/lib/jenkins/config.xml').grep(/GithubSecurityRealm/).any?
-      node.run_state[:jenkins_username] = secrets['git']['ibmz_ci']['user'] # ~FC001
-      node.run_state[:jenkins_password] = secrets['git']['ibmz_ci']['token'] # ~FC001
-    end
-  end
-end
-
```

Comment:
Why was this removed?

Reply:
This is a duplicate of the `ruby_block` in line 172.

Reply:
I'd prefer if you removed the block at line 172 and keep the one at the top.

FYI: This is also happening in the powerci recipe.